tests/unit/data/demo_settings.yaml

```
@@ -0,0 +1,2 @@
+development:
+    SECRET_KEY: "dev.key"
```

```suggestion
SECRET_KEY: "dev.key"
```

With the suggested change above the `development` here is not needed.